Dockerfile.rocm

```diff
@@ -37,10 +61,11 @@ RUN mkdir libs \
     && cd libs \
     && git clone https://github.com/ROCmSoftwarePlatform/flash-attention.git \
     && cd flash-attention \
-    && git checkout 3d2b6f5 \
+    && git checkout ${FA_BRANCH} \
     && git submodule update --init \
-    && export GPU_ARCHS=$(/opt/rocm/llvm/bin/amdgpu-offload-arch) \
-    && patch /opt/conda/envs/py_3.10/lib/python3.10/site-packages/torch/utils/hipify/hipify_python.py hipify_patch.patch \
```

Comment:
you may still need this line, but it'll have to be the python version of the docker image (3.9 for rocm 6.0's image case)

Reply:
ideally, the later 6.0 will not need this

Reply:
I agree with Doug, the hipify_pytorch.py path will change base on conda env py_3.9 or py_3.10.
And regarding regarding the hipify patch is it required for all versions of pyTorch ? for 2.1v or 2.2v also?